README.md

```diff
@@ -1,4 +1,6 @@
 <a name="readme-top"></a>
+<img src="200w.gif" alt="logo" width="140"  height="auto" />
+<br/>
```

Comment:
- [ ] [OPTIONAL] Please consider renaming the project name in the PROJECT DESCRIPTION section of your README with a more appropriate title. It is currently named **ToDo List** 👍 
![image](https://user-images.githubusercontent.com/46839548/230707440-5c79629f-0516-46bd-9253-821c8801a02d.png)